test/fast/Unit tests/nvm_print_color_code

```diff
@@ -16,5 +16,5 @@ set +x # needed for stderr
 OUTPUT="$(nvm_print_color_code q 2>&1)" ||:
 set -x
 echo "OUTPUT WAS SET TO: $OUTPUT"
-EXPECTED_OUTPUT="Invalid color code"
+EXPECTED_OUTPUT="Invalid color code: q"
```

Comment:
```suggestion
EXPECTED_OUTPUT="Invalid color code"
```

Reply:
This would revert an intentional change to ease debugging.